content/en/security_platform/cloud_siem/cloud_siem_investigator.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Cloud SIEM Investigator
+kind: documentation
+aliases:
+  - /security_platform/cloud_siem/cloud_security_investigator/
+further_reading:
+- link: "/cloud_siem/explorer/"
+  tag: "Documentation"
+  text: "Learn about the Security Signals Explorer"
+---
+
+{{< beta-callout url="#" btn_hidden="true">}}
+  The Cloud SIEM Investigator is in beta.
+{{< /beta-callout >}} 
+
+<div class="alert alert-warning">Cloud SIEM Investigator currently only supports AWS CloudTrail logs.</div>
+
+## Overview
+
+When a security signal alerts on suspicious activity by a user or a resource, common questions asked as part of the investigation could include:
+
+- Is the user accessing other accounts? 
+- What other actions did the user take around that specific time frame? 
+- What are all the actions taken on a resource by the user?
+- What users have interacted with this resource?
+
+For example, suppose you receive a security signal that someone changed the configuration of an AWS S3 bucket so that it is accessible by everyone, but the action was taken by an assumed role. To investigate, look into who took the action and what other activities they did recently, as that could indicate compromised credentials.
+
+The Cloud SIEM Investigator provides a graphical interface for you to pivot from one affected entity to another, so that you can see user behavior and its impact on your environment.
+
+
+## Visualize and investigate the activity
```

Comment:
```suggestion
## Visualize and investigate user activity
```

Reply:
Will this as is because it could be user or service activity.